.github/workflows/rebase-cmd-dispatch.yml

```diff
@@ -0,0 +1,19 @@
+# On any comment, it will look for '/rebase' in the comment body and in case of hit, it dispatches rebase cmd
+# with event type 'rebase-command' which triggers 'rebase-command` WF that performs the rebase operation.
```

Comment:
WF?

Reply:
this doesn't explain the relation between `rebase-cmd-dispatch` and `rebase-cmd`, I still don't understand what it is